src/Docker/Runner/StateFile.php

```diff
@@ -16,7 +17,13 @@
 
 class StateFile
 {
-    const NAMESPACE_PREFIX = 'component';
+    const COMPONENT_NAMESPACE_PREFIX = 'component';
```

Comment:
myslim, ze by stacilo jen `COMPONENT_NAMESPACE`, zadnej prefix to vlastne neni

Reply:
jasně, nebo možná `NAMESPACE_COMPONENT`?

Reply:
klidne